python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         ndjson_normalize_sep: str = "_",
         use_native_support: bool = True,
         native_support_kwargs: dict | None = None,
+        databricks_options: dict | None = None,
```

Comment:
Let's avoid specific tech kwarg in LoadFile constructor

Reply:
@kaxil let's discuss in our 1:1 I think it might honestly make sense to do it this way.